dialogue/TownNPCs/Olive/InitialInteractions/Pistachio/Pistachio_Olive_First4.json

```diff
@@ -6,11 +6,16 @@
       "Stachi, I really do suggest you think this\nthrough more.",
       "Take your time.\nI'll always be here for you!"
     ],
+    "pl": [
+      "Stacju, proponuję, abyś pomyślała nad tym troszkę więcej.",
```

Comment:
I'm actully not sure if it's not better to keep the english version here instead of translating it, because "Stachi" sounds better compared to "Stacjo"

Reply:
But I'm really not sure about this one